README.md

```diff
@@ -37,9 +37,10 @@ These programs and resources are required to install and run Lisk Explorer
 
   `sudo npm install -g grunt`
 
-- Forever (<https://github.com/foreverjs/forever>) -- Forever manages the node processes for Lisk Explorer
+- PM2 (https://github.com/Unitech/pm2) -- PM2 manages the node process for Lisk Explorer and handles log rotation (Highly Recommended)
 
-  `sudo npm install -g forever`
+  `sudo npm install -g pm2`
+  `pm2 install pm2-logrotate`
```

Comment:
Would be nice to describe how to configure `pm2-logrotate` correctly (`max_size`, `compress`, etc.)